src/assets/css/utils/_themes.scss

```diff
@@ -0,0 +1,44 @@
+$themes: (
+  light: (
+    elevation0: #f8fbfe,
+    elevation1: #ffff,
+    elevation2: #ffff,
+    elevation3: #ffff,
+    elevation4: #ffff,
+    elevation5: #ffff,
```

Comment:
What's the difference between elevation 1 till 5?

Reply:
Do we also need 5 different elevations? Or how much do we actually use. Same goes for dark theme a few lines below.